common/scala/src/main/scala/whisk/core/entity/WhiskActivation.scala

```diff
@@ -119,4 +124,28 @@ object WhiskActivation
   // Caching activations doesn't make much sense in the common case as usually,
   // an activation is only asked for once.
   override val cacheEnabled = false
+
+  /**
+   * Queries datastore for activation records which have an entity name matching the
+   * given parameter.
+   *
+   * @return list of records as JSON object if docs parameter is false, as Left
+   *         and a list of the WhiskActivations if including the full record, as Right
+   */
+  def listActivationsMatchingName(db: ArtifactStore[WhiskActivation],
+                                  namespace: EntityPath,
+                                  name: EntityName,
+                                  skip: Int,
+                                  limit: Int,
+                                  docs: Boolean = false,
```

Comment:
What do think about renaming this to `includeDocs`.
Then it would be consistent with `listCollectionInNamespace`.

Reply:
yes. I missed it.

Reply:
Do you mind if I address this in the next dependent pr https://github.com/apache/incubator-openwhisk/pull/2760? There’s another docs to rename there.

Reply:
No problem.